src/execute.js

```diff
@@ -108,12 +108,16 @@ export function buildRequest({
         value = parameters
       }
 
-      value = parameter && parameter.name && parameters[parameter.name]
+      value = (parameter && parameter.name && parameters[parameter.name]) || value
```

Comment:
The if statement above was dead code because of this line. This change allows you to use the parameters object as the value of a body parameter (parameters = {mydata: ''}) instead of having to have parameters be: {body_param_name: {mydata: ''}}